content/stopping-systemd-in-openshift.md

```diff
@@ -24,9 +23,15 @@ limitation in Kubernetes by using container lifecycle hooks.
 > [kcli](https://github.com/karmab/kcli) or
 > [Code Ready Containers](https://github.com/code-ready/crc).
 
+**UPDATES**:
+
+- This is happening in Openshift but it will be fiex inin 4.10 (verified on
+  Openshift 4.10.0-ci-20220107).
+- Here is the change at cri-o that fix this situation: 
```

Comment:
missing link?

Reply:
It should be https://github.com/cri-o/cri-o/pull/5366 ; updating, thanks